pytket/extensions/qiskit/backends/ibm.py

```diff
@@ -346,7 +346,45 @@ def required_predicates(self) -> List[Predicate]:
             ] + predicates
         return predicates
 
-    def default_compilation_pass(self, optimisation_level: int = 2) -> BasePass:
+    def default_compilation_pass(
+        self, optimisation_level: int = 2, placement_options: Optional[Dict] = None
+    ) -> BasePass:
+        """
+        A suggested compilation pass that will will, if possible, produce an equivalent
+        circuit suitable for running on this backend.
+
+        At a minimum it will ensure that compatible gates are used and that all two-
+        qubit interactions are compatible with the backend's qubit architecture. At
+        higher optimisation levels, further optimisations may be applied.
+
+        This is a an abstract method which is implemented in the backend itself, and so
+        is tailored to the backend's requirements.
+
+        The default_compilation_pass for the IBMQBackend, IBMQEmulatorBackend and the
+        Aer simulators support an optional placement_options dictionary containing
+        arguments to override the default settings in NoiseAwarePlacement.
+
+        :param optimisation_level: The level of optimisation to perform during
+            compilation.
+
+            - Level 0 does the minimum required to solves the device constraints,
+              without any optimisation.
+            - Level 1 additionally performs some light optimisations.
+            - Level 2 (the default) adds more computationally intensive optimisations
+              that should give the best results from execution.
+
+        :type optimisation_level: int, optional
+
+        :param placement_options: Optional argument allowing the user to override
+          the default settings in
+          `NoiseAwarePlacement
+          <https://cqcl.github.io/tket/pytket/api/placement.html#pytket.placement.NoiseAwarePlacement>`_.
+          Options:
+          maximum_matches, timeout, maximum_pattern_gates, maximum_pattern_depth.
```

Comment:
Listing them here means we're liable to get out of sync if the `NoiseAwarePlacement` options are extended -- the link should be enough.

Reply:
Seems reasonable, done.